### PR TITLE
edits for the 2-week project

### PR DESCRIPTION
**Pull request by datn**
Alll my revisions for you!

### CODE REVIEW

File: source/install.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+   `docker-compose up -d`
+
+   (To view the continuous output, you can run `docker-compose logs -f`)
+
+4. Verify that all the services have started. 
+    1. Run `docker-compose ps`. You should see eight images in an “Up” state. 
+    2. Check the logs for success or errors -- for example, using this command:
+
+    `docker-compose logs --tail="20" -t`
```
Comment (wiese) [inline source/install.md:38]: This needs an extra level of indentation (not sure if it will work out of the box, though).

**Comment by wiese** [comment]
I rebased this in #3 and added a few follow-ups in #4

File: source/install.md (hunk 1 above)
Comment (wiese) [reply source/install.md:38]: Did that in 137727a

**Comment by addshore** [comment]
Going to close this one and we will merge https://github.com/wmde/wikiba.se/pull/3

**Pull request closed without merging**